jenkins_pipelines/environments/common/pipeline-pull-request.groovy

```diff
@@ -15,6 +15,8 @@ def run(params) {
         terraform_init = true
         rake_namespace = 'cucumber'
         env_number = 6
+        repo_dir = '/home/jenkins/jenkins-build/workspace/'
```

Comment:
Uhmmm better uses $WORKSPACE variable and from there use something like `$(cd ${WORKSPACE}/.. && pwd)` if you want to avoid the folder of the job itself.
See this doc section: https://www.jenkins.io/doc/book/pipeline/jenkinsfile/#using-environment-variables

Reply:
Not sure ... I want to use /home/jenkins/jenkins-build/workspace/ because this is what is configured in apache2